GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -8,6 +8,10 @@ namespace gdjs {
     _pixiContainer: any;
     _debugDraw: PIXI.Graphics | null = null;
     _profilerText: PIXI.Text | null = null;
+    _renderedObjectsPoints: Array<{
```

Comment:
Let's rename this to `_debugDrawRenderedObjectsPoints`. It's long and verbose, but clear that this is something related to `_debugDraw` and not needed otherwise :)

Reply:
Also, this should not be an array, but an associative map, i.e: an object. `Record<number, { ...}>`